superbench/benchmarks/micro_benchmarks/memory_bw_latency_performance.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module for running the Intel MLC tool to measure memory bandwidth and latency."""
+
+import os
+from pathlib import Path
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class MemBwLatencyBenchmark(MicroBenchmarkWithInvoke):
+    """The Memory bandwidth and latency benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'mlc'
+        self.__support_mlc_commands = ['bandwidth_matrix','latency_matrix','max_bandwidth']
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument(
+                '--tests',
+                type=str,
+                nargs='+',
+                default=["bandwidth_matrix"],
+                required=False,
```

Comment:
When this value is changed to `True` I cannot run the example. @guoshzhao could you please check to see what could be wrong?

Reply:
I think we can set it as `False` because we have default value.

Reply:
@hpourreza If we set one argument as `required=True`, we should give the value when register the benchmark:
`BenchmarkRegistry.register_benchmark('memory-bw-latency-benchmark', MemBwLatencyBenchmark, parameters='--tests bandwidth_matrix')`

Agree with Peng, since there already has default value, we can just set `required=False`.

Reply:
Got it. Makes sense. I will leave it as False with a default value.